src/containers/DataPackageView.js

```diff
@@ -18,8 +19,12 @@ export class DataPackageViewContainer extends React.Component {
   }
 
   componentDidMount(){
-    let url = "https://raw.githubusercontent.com/anuveyatsu/test_data/master/datapackage.json";
+    let baseUrl = "https://bits.staging.datapackaged.com/metadata/";
+    let url = baseUrl + this.props.publisherName + "/" + this.props.packageName + "/_v/latest/datapackage.json";
+    // let url = "https://raw.githubusercontent.com/anuveyatsu/test_data/master/datapackage.json";
     this.props.dataPackageActions.getDataPackage(url);
+    this.props.dataPackageActions
+      .getDataPackageMetaData(this.props.publisherName, this.props.packageName);
```

Comment:
We do not know what is the name of the readme file. e.g. ```README/readme/ReadME```
How many http call should we do.